docs/getting-started/requirements.md

```diff
@@ -98,7 +98,23 @@ Prowler will follow the same credentials search as [Google authentication librar
 2. [User credentials set up by using the Google Cloud CLI](https://cloud.google.com/docs/authentication/application-default-credentials#personal)
 3. [The attached service account, returned by the metadata server](https://cloud.google.com/docs/authentication/application-default-credentials#attached-sa)
 
-Those credentials must be associated to a user or service account with proper permissions to do all checks. To make sure, add the `Viewer` role to the member associated with the credentials.
+### Needed permissions
+
+Prowler for Google Cloud needs the following permissions to be set:
+
+- **Viewer (`roles/viewer`) IAM role**: granted at the project / folder / org level in order to scan the target projects
+
+- **Project level settings**: you need to have at least one project with the below settings:
+    - Identity and Access Management (IAM) API (`iam.googleapis.com`) enabled by either using the
```

Comment:
Is not this API enabled by default when you use the IAM service for the first time in the console?

Reply:
No, not even in this case (I've checked on a new Google Account and project and also on a new project as part of an org). Furthermore, most of the times you're not granted such highly permissive role for doing a security review across the org.

Reply:
I am not talking about the role but about the IAM API in the project level settings.